### DISCUSSION TITLE
Button Icon for lucide names with dashes

### DISCUSSION THREAD

**Discussion by shaneallen001** [mProjectsCode/obsidian-meta-bind-plugin#376] [Q&A]
I am creating buttons and having problems filling out the Icon field.
I seem to only be able to use icons that are single words "user".
If I try "user-check", for example, there's no icon.

I've tried quotes, no quotes, single quotes, no dashes, dashes, periods, spaces... nothing.

How do I specify a lucide icon like user-pen?

**Comment by mProjectsCode** [comment]
Only icons that obsidian includes are available, and afaik Obsidian uses an older version of Lucide, so that might be the problem.
